addons/circulation/functions/fnc_drawArterial.sqf

```diff
@@ -0,0 +1,29 @@
+#include "..\script_component.hpp"
+/*
+ * Author: Battlekeeper, modified by YetheSamartaka, Mazinski
+ * Refactored by Slatery
+ * Handles drawing of arterial blood
+ *
+ * Arguments:
+ * 0: Medic <OBJECT>
+ * 1: Patient <OBJECT>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [medic, patient] call kat_circulation_fnc_drawArterial;
+ *
+ * Public: No
+ */
+
+params ["_medic", "_patient"];
+
+private _bloodGas = GET_BLOOD_GAS(_patient);
+
+GVAR(sampleCounter) = if (GVAR(sampleCounter) == 20) then { 1 } else { GVAR(sampleCounter) + 1 };
```

Comment:
cant you use select here? tho its more as suggestion because i remember select is faster for those type of operations and its just micro optimisation at this moment xD
```suggestion
GVAR(sampleCounter) = [GVAR(sampleCounter) +1, 1] select (GVAR(sampleCounter) ==20);
```